public/video-ui/src/components/Video/Display.js

```diff
@@ -166,18 +181,22 @@ class VideoDisplay extends React.Component {
                   {this.renderEditButton('youtubeEditable')}
                 </div>
                 <YoutubeMetaData
-                  component={YoutubeMetaData}
                   video={this.props.video || {}}
-                  saveVideo={this.saveVideo}
                   updateVideo={this.updateVideo}
-                  disableStatusEditing={this.cannotEditStatus()}
                   editable={this.props.editState.youtubeEditable}
+                  updateFormErrors={this.updateYoutubeFormErrors}
                 />
               </div>
               <div className="video__detailbox">
                 <div className="video__detailbox__header__container">
                   <header className="video__detailbox__header">Poster Image</header>
-                  <GridImageSelect editState={this.props.editState} video={this.props.video || {}} updateVideo={this.saveAndUpdateVideo} gridUrl={this.props.config.gridUrl} createMode={false}/>
+                  <GridImageSelect
+                    editState={this.props.editState}
+                    video={this.props.video || {}}
```

Comment:
as above, do we need this property still (instead of setting the value as video.posterImage?)